### DISCUSSION TITLE
Flowise generates new Open assistant (new threadID) even when passing correctly the chatId

### DISCUSSION THREAD

**Discussion by Rudi-AI-Oficial** [FlowiseAI/Flowise#1510] [Q&A]
As we know, chatId is specifically for OpenAI assistants. It should keep the same openai thread session alive. 

The problem is:  I pass the chatId the way is explained in documentation, but even though, flowise creates a new threadId in OpenAi assistant. Can someone help me to figure out why?

Context: flowise is running in render on the cloud. Those pics are from Make.com where I have a curl calling using the Http module to talk with flowise. 

Objective: I wanna keep using the same threadId from the OpenAI Assistant so the assistant does not forget the previous messages.
![Screenshot 2024-01-10 145001](https://github.com/FlowiseAI/Flowise/assets/155680808/e81dcd44-bafe-454e-aa7b-b7c2f60fa77e)
![Screenshot 2024-01-10 125450](https://github.com/FlowiseAI/Flowise/assets/155680808/0e831c9e-cbd2-42d4-b279-4e69ab9a59be)

**Comment by Rudi-AI-Oficial** [comment]
I noticed that if I call the API using python it works.... it keeps the threadID for the same userID.
But something goes wrong when calling through Make.com

**Reply by HenryHengZJ** [reply]
@Rudi-AI-Oficial I tested on Postman it works fine.

1.) Have a OpenAI Assistant flow:
![image](https://github.com/FlowiseAI/Flowise/assets/26460777/90e49728-7190-4d35-acf9-bda9b40b0afb)

2.) Send 1st API:
![1](https://github.com/FlowiseAI/Flowise/assets/26460777/755a34b5-b268-4a2f-b620-d841f10c248f)

3.) Send 2nd API:
![2](https://github.com/FlowiseAI/Flowise/assets/26460777/f1be46ef-ddfb-497d-b729-62524e66c198)

You can see that it remembers the conversation

**Comment by MarecoX** [comment]
I'm trying to do the same with n8n, using POST to maintain the same Thread, however, so far without success, have you found any solution?
